src/main/java/com/hackclub/hccore/listeners/PlayerListener.java

```diff
@@ -68,7 +68,11 @@ public void onPlayerJoin(final PlayerJoinEvent event) {
         event.getPlayer().resetTitle();
         event.setJoinMessage(ChatColor.YELLOW
                 + ChatColor.stripColor(event.getPlayer().getDisplayName()) + " joined the game");
-    }
+
+        if (event.getPlayer().hasPlayedBefore() == false) {
+            event.getPlayer().performCommand("tutorial");
```

Comment:
You should make a function called `startTutorialFlow` that takes a player as the parameter. This feels a little hacky lol.